exercises/phone-number/phone_number_test.exs

```diff
@@ -38,6 +38,16 @@ defmodule PhoneTest do
   end
 
   @tag :pending
+  test "invalid when proper number of digits but letters mixed in and length is 10" do
```

Comment:
This description and the next one are a little misleading since there aren't the `proper number of digits`. We'd need 10 digits, but there are only 5 here.

Something like `invalid with correct number of characters but some are letters` might be more clear.